### ISSUE TITLE
Discussion: Audio Inputs

### ISSUE THREAD

**Issue by billvanloo** [NorthCoastModularCollective/BridgeSystemOnRampPrototype#8]
We've currently included a pair of audio inputs on V2 of the prototype. We need to decide how these will work. A few options:

1. [ ] Audio inputs act as an aux input, being mixed with the system's audio output and passed to the outputs. Essentially, they serve as a way to get another sound source mixed down to the output (ex: phone, a second Bridge System, etc).
2. [ ] Oscillator replacement. In this scenario, the audio input would replace the oscillator, and would then get sent through the rest of the signal path (distortion, filter, etc).

There could be other options, too. One note: there's currently no audio level control associated with the audio inputs. We also have a stereo input, so the inputs could potentially be treated as two separate mono inputs, I think (@joebauer is this correct?).

**Comment by joebauer** [comment]
Yes, the inputs could be treated as two separate mono inputs. Additionally, (just in terms of hardware capabilities) we can do 2 audio inputs at either line level or 2 at mic level. I don't think we can run all 4 simultaneously, but we can switch between which ones (mic level or line level) are being used. 

I think I'm interested in exploring the idea of using one mono audio input as an aux in so you can chain a few On Ramps (maybe have an attenuator knob (electronics circuit-based) on that input for some volume control). Then use another as an "oscillator" source. For that one we could put a knob in which cross fades between the oscillator volume and the audio input. On one side it would be only oscillator, on the other only audio input, at 12 o'clock it would be both equally.
